pkg/scheduler/framework/plugins/tainttoleration/taint_toleration.go

```diff
@@ -56,7 +56,7 @@ func (pl *TaintToleration) Name() string {
 // failed by this plugin schedulable.
 func (pl *TaintToleration) EventsToRegister() []framework.ClusterEvent {
 	return []framework.ClusterEvent{
-		{Resource: framework.Node, ActionType: framework.Add | framework.UpdateNodeTaint},
```

Comment:
This one too

Reply:
yeah, I left that out just because we need to patch this.